pkg/rpm/database.go

```diff
@@ -41,6 +41,9 @@ const (
 	// databaseDir is the directory where the RPM database is expected to be in
 	// the container filesystem.
 	databaseDir = "var/lib/rpm"
+	// rhcosDbDir is an alternative to databaseDir, as RHCOS saves its RPM DB in
+	// a different path.
+	rhcosDbDir = "usr/share/rpm"
```

Comment:
1. What about symlink story? I think it is worth mentioning it here in comments. More precisely, my question is about `var/lib/rpm` being a symlink to `usr/share/rpm`. Won't we end up with duplicate entries due to the same content getting discovered from two seemingly different paths?

2. Also, [nit], previous items were separated by a blank line.